src/lib.rs

```diff
@@ -79,7 +79,6 @@ mod macros;
 // Public API
 // ############################################################################
 pub use self::{
-    consensus::{AccumulationError, Proof, ProofShare, SignatureAccumulator},
     error::RoutingError,
```

Comment:
I suspect this crate should have it's own error type and not RotuingError. If we use err_derive then it will autmatically impl From for us.